internal/provider/kubernetes/config/envoy-gateway/deploy_and_ns.yaml

```diff
@@ -28,8 +28,8 @@ spec:
         runAsNonRoot: true
       containers:
       - name: envoy-gateway
-        image: envoyproxy/gateway-dev:latest
-        imagePullPolicy: Always
+        image: docker.io/envoyproxy/gateway-dev:981e90e
```

Comment:
lets revert ?

Reply:
Will do. I created https://github.com/envoyproxy/gateway/issues/351 so this type of change doesn't get accidentally merged.